chaincode/ledger_db.go

```diff
@@ -443,14 +443,14 @@ func (db *LedgerDB) GetNode(key string) (Node, error) {
 // SendTuplesEvent sends an event with updated tuples if there is any
 // Only one event can be sent per transaction
 func (db *LedgerDB) SendTuplesEvent() error {
```

Comment:
Should be renamed to `SendEvent()` (and update its documentation)?

Reply:
You always miss one, don't you... Will do, thanks for the review

Reply:
Yup :)